intranet/apps/eighth/views/attendance.py

```diff
@@ -300,6 +300,29 @@ def take_attendance_view(request, scheduled_activity_id):
 
         present_user_ids = list(request.POST.keys())
 
+        if request.FILES["attendance"]:
```

Comment:
Darn, I'm a fool; this would `KeyError` if `"attendance"` doesn't exist (aka a file wasn't uploaded).

PR incoming